app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+
+import { Formik, Form } from "formik";
+import { Check } from "neetoicons";
+import { Button, Pane } from "neetoui";
+import { Input, Textarea } from "neetoui/formik";
+
+import notesApi from "apis/notes";
+
+import { CONTACT_FORM_VALIDATION_SCHEMA } from "../constants";
+
+const ContactForm = ({ onClose, refetch, note, isEdit }) => {
+  const [submitted, setSubmitted] = useState(false);
+
+  const handleSubmit = async values => {
+    try {
+      if (isEdit) {
+        await notesApi.update(note.id, values);
+      } else {
+        await notesApi.create(values);
+      }
+      refetch();
+      onClose();
+    } catch (err) {
+      logger.error(err);
+    }
+  };
+  // TODO: This form is to be completed in the next PR (one involving creation of new contact form)
+  return (
+    <Formik
+      initialValues={note}
+      validateOnBlur={submitted}
+      validateOnChange={submitted}
+      validationSchema={CONTACT_FORM_VALIDATION_SCHEMA}
+      onSubmit={handleSubmit}
+    >
+      {({ isSubmitting }) => (
+        <Form className="w-full">
+          <Pane.Body className="space-y-6">
+            <Input
+              required
+              className="w-full flex-grow-0"
+              label="Title"
+              name="title"
+              placeholder="Enter note title"
+            />
+            <Textarea
+              required
+              className="w-full flex-grow-0"
+              label="Description"
+              name="description"
+              placeholder="Enter note description"
+              rows={2}
+            />
+          </Pane.Body>
+          <Pane.Footer>
+            <Button
+              className="mr-3"
+              disabled={isSubmitting}
+              icon={Check}
+              label={isEdit ? "Update" : "Save Changes"}
+              loading={isSubmitting}
+              size="large"
+              style="primary"
+              type="submit"
+              onClick={() => setSubmitted(true)}
+            />
+            <Button
+              label="Cancel"
+              size="large"
+              style="text"
+              onClick={onClose}
+            />
```

Comment:
Go through the Slack thread. I had pointed out about `reset` type.

Reply:
Thanks, Yedhin for pointing this out. I have rectified this.